script/Ciel/Day 3/03_01_CIEL03_3_3C.txt

```diff
@@ -301,17 +301,18 @@ I couldn't give her a single satisfactory reply, but I felt sorry and happy at t
 [sha:2662d66caa3f0700648f479131727c5b50748245]{
 -- Page 266, Offset 5529.
 -- 不安で眠れないのなら、この先どう償って生きていくかを考えればいいんです。
- If you can't sleep because of anxiety, you should think about how you can atone for what you've done.
+If you can't sleep because of anxiety, you should think about how you can atone for what you've done.
 }
 [sha:153edb1c4d01fc1701923281ccf35643fc53c30f]{
 -- Page 266, Offset 5530.
 -- それはすごく難しいことですから、そのうちあたまがパンクしてするっと眠れちゃいます」
- It's a very difficult thing to do, so your mind will eventually go blank and you'll be able to sleep easily."
+// B: Catholic sleep hacks
```

Comment:
i tried this once and jesus talked to me in my sleep. 10/10 would recommend

Reply:
🙏 Thank you Ciel for these Sleep Hacks 🙏